esmvalcore/preprocessor/_time.py

```diff
@@ -636,25 +637,32 @@ def anomalies(cube,
                                              seasons=seasons)
             cube = cube / cube_stddev
             cube.units = '1'
-        return cube
-
-    cube = _compute_anomalies(cube, reference, period, seasons)
-
-    # standardize the results if requested
-    if standardize:
-        cube_stddev = climate_statistics(cube,
-                                         operator='std_dev',
-                                         period=period)
-        tdim = cube.coord_dims('time')[0]
-        reps = cube.shape[tdim] / cube_stddev.shape[tdim]
-        if not reps % 1 == 0:
-            raise ValueError(
```

Comment:
so I was trying to write a test for this statement here a la:
```python
def test_standardized_anomalies_fail():
    """Test standardized ``anomalies`` failure due to shape mismatch."""
    cube = make_map_data(number_years=0.01)
    cube = cube[0:2]
    print(cube.coord("time"))
    with pytest.raises(ValueError):
        anomalies(cube, period="season", standardize=True)
```
which doesn't fail even if the cube's time axis is made up of two single points (two days) - surely this is something odd isn't it? Computing seasonal anomalies on two days should probably either fail or give the user a BIG warning data is totally unreliable?